src/backend/ethereum/geth.ts

```diff
@@ -45,7 +45,7 @@ export default class Geth {
     return {command: 'geth', options: ['--datadir', datadir, 'init', genesis]};
   }
 
-  public static cleanup(datadir: string, callback?: (error: Error) => void) {
+  public static clean(datadir: string, callback?: (error: Error) => void) {
```

Comment:
We might want to document the choice between async remove and sync remove here. Usually when a method takes an optional callback, it's still asynchronous, it's just there won't be any notification of completing the task.

Reply:
yeah fair enough. i don't think we need async for now - i'm gonna drop it and if we need it later we can add it then.